packages/pn-commons/src/components/TitleBox.tsx

```diff
@@ -52,7 +52,9 @@ const TitleBox: React.FC<Props> = ({
           role="heading"
           variant={variantTitle}
           display="inline-block"
-          sx={{ verticalAlign: 'middle' }}
+          sx={{ verticalAlign: 'middle', maxWidth:{ xs:'70vw', lg:'40vw' },
```

Comment:
Using widths in this cases is not a good practice because it depends on the resolution and the screen size.
Try to use a css rules that doesn't depend on screen size or resolution

TIPS: check overflow-wrap

Reply:
done